### DISCUSSION TITLE
[1448030]Press Any Button

### DISCUSSION THREAD

**Discussion by Porridge10010** [pjy612/SteamManifestCache#194] [Share]
appid:1448030
keys:
```
1448035;e47d93961f72366198492d6dd2889cf4a9e9c40c50449abd2f8f86466bf6a74f
```
[Press Any Button.zip](https://github.com/pjy612/SteamManifestCache/files/13785897/Press.Any.Button.zip)

code:14F8BD95F99498F8
email:1994392644@qq.com

**Comment by pjy612** [comment]
emmm 太冷门和便宜了吧。。。

**Comment by Porridge10010** [comment]
确实。。。。但是我游戏库里实在找不出冷门但高价的，不好意思哈。



---Original---
From: ***@***.***&gt;
Date: Thu, Dec 28, 2023 20:26 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [pjy612/SteamManifestCache] [1448030]Press Any Button (Discussion#194)




 
emmm 太冷门和便宜了吧。。。

**Reply by pjy612** [reply]
没事，查收

**Comment by Porridge10010** [comment]
哦对了我还想问一下tg群组在哪加入，我找了半天没找着




1994392644
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "pjy612/SteamManifestCache"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月28日(星期四) 晚上10:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [pjy612/SteamManifestCache] [1448030]Press Any Button (Discussion #194)





 
没事，查收

**Reply by pjy612** [reply]
steamtools 官网上面看 入口
